### DISCUSSION TITLE
Stream equivalent for iOS

### DISCUSSION THREAD

**Discussion by anilmaddala** [google/filament#6493] [Q&A]
I see good examples and documentation and examples for using Stream in Android.

However I couldn't find similar support for iOS. Is there something similar for iOS?

**Comment by bejado** [comment]
The closet thing on iOS would be support for `CVPixelBufferRef` textures. See [`Texture::setExternalImage`](https://github.com/google/filament/blob/2a049d13244a44077d61c01e0a3a139d63297459/filament/include/filament/Texture.h#L399).
